moveit_setup_assistant/include/moveit/setup_assistant/tools/moveit_config_data.h

```diff
@@ -143,15 +145,15 @@ class GenericParameter
 
   void setName(std::string name)
   {
-    name_ = name;
+    name_ = std::move(name);
   };
   void setValue(std::string value)
   {
-    value_ = value;
+    value_ = std::move(value);
   };
   void setComment(std::string comment)
   {
-    comment_ = comment;
+    comment_ = std::move(comment);
```

Comment:
[performance] Because you are manually preventing copy elision optimization here this is a case where using std::move will create worse performance.